### ISSUE TITLE
Death Causes IHME vs IFs

### ISSUE THREAD

**Issue by quciet** [PardeeCenterDU/IFs-Issues-Tracking#226]
This is a low-priority issue since no project is currently involved, yet this is important since we haven't updated these tables for a long time. This is related to Death Causes in IFs.mdb.

In IFs, we have 15 death causes, corresponding to level-2 death causes in IHME. However, IHME has 22 level-2 death causes, out of which 15 can be matched directly to IFs causes. Thus, there are 7 level-2 causes left from IHME, e.g., maternal diseases and nutritional diseases. Two questions,
1. When would be the best time to make an update? How big of an impact it will have?
2. If we are to update the table, do we want to make an **upgrade** so that we can include more disease types easily moving forward?

**Comment by quciet** [comment]
Change of plan after the modeling pod meeting on Sep 19th, 2022. Here are my takeaways- 

1. If we do have a project that requires us to take a deeper dive into specific disease types, we will make an update. Until then, let's try to include all other new types into Other CD or Other NCD.
2. We will take a look and see how many tables in IFs.mdb or IFsHistSeries will need to be updated for each IHME release, and make a wiki page for the process.

Feel free to add more if I'm missing out on some information. 

Thanks,
Yutang

**Comment by quciet** [comment]
Regarding point 2 above, based on the email from José, here are some tables we would consider-

HealthDetailedDeathsCtry (currently under update)

HealthYLDComMatPerCohortCountryMale/Female
HealthYLDNonComDisCohortCountryMale/Female
HealthYLDInjuriesCohortCountryMale/Female

HealthGBDYLD (This is the one used to build the Ratio YLD/YLL, this one uses WHO Regional groups, and is used when there’s no data for the first 6)

HealthMortRatios (Used to Load Ratios of Mortality Subtypes to Mortality types)

HealthCostByType

HealthProxDrivRRs (low priority)

**Comment by quciet** [comment]
Another context regarding these tables-

Cohort0 is Infants only (younger than 1 year)
Cohort1 is 1-4.
Cohort21 is 100+.
All others are 5 year age groups

**Comment by collinmeisel** [comment]
Is this an annual update? Curious which, if any, of these variables touches the preprocessor.

Thanks!
Collin

**Comment by quciet** [comment]
> Is this an annual update? Curious which, if any, of these variables touches the preprocessor.
> 
> Thanks! Collin

I think these are rather global parameters in IFs.db. So yes, they will change model behaviors. My understanding is that they behave differently than preprocessors in the IFsHistSeries.db. 

In terms of the update schedule, those are not included in the calendar yet. There are many such tables in IFs.db that I will have to understand their context before updating them.

Thanks,
Yutang

**Comment by quciet** [comment]
Currently, we have an updated death cause concordance table. 
[IHMF IFs disease concordance 20240221.xlsx](https://github.com/PardeeCenterDU/IFs-Issues-Tracking/files/14364737/IHMF.IFs.disease.concordance.20240221.xlsx)

And this HealthDetailedDeathsCtry is being updated and will be included in data update schedule (every Feb).

**Comment by quciet** [comment]
During this year's update (Feb 2025), we found out that IHME added a new level-1 death cause- COVID. Because they extended data from 2019 to 2021. We need to discuss how we are going to integrate it into our database before making the update. 

<img width="1061" alt="Image" src="https://github.com/user-attachments/assets/7d48792e-e86a-4f00-aa88-d61ac4f8e8a3" />

**Comment by collinmeisel** [comment]
Per modeling pod: Let's *not* create a new COVID-19 category. Instead, lump into whatever the most appropriate broader category is.

That said: Something to watch is how the COVID death spike in 2020-2021 affects the forecasts thereafter.

**Comment by quciet** [comment]
<img width="1053" alt="Image" src="https://github.com/user-attachments/assets/434862c0-7605-4edf-81ad-dffb459fc765" />

More note on this: 
In IHME, COVID-19 is directly put under the cause of "Respiratory infections and tuberculosis" as a level-3 cause, in parallel to other level-3 causes such as "Lower respiratory infections". Additionally, IHME added a new level-1 cause called "other COVID-19 related outcome" that is not categorized under any of the three level-1 causes. Thus, 

<img width="381" alt="Image" src="https://github.com/user-attachments/assets/61ad699e-0203-42e6-be44-9787f7fdc450" />

1. We can probably put "other COVID-19 related outcomes" under "respiratory infection", but this needs a consensus from the team.
2. Even if we ignore this newly added level-1 cause, in the updated data, we should see a huge spike in the "Respiratory Infection" in IFs. We need to investigate how it will drive further model changes.

**Comment by collinmeisel** [comment]
> <img alt="Image" width="1053" src="https://private-user-images.githubusercontent.com/34465524/411702368-434862c0-7605-4edf-81ad-dffb459fc765.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkyMTc3NDIsIm5iZiI6MTczOTIxNzQ0MiwicGF0aCI6Ii8zNDQ2NTUyNC80MTE3MDIzNjgtNDM0ODYyYzAtNzYwNS00ZWRmLTgxYWQtZGZmYjQ1OWZjNzY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjEwVDE5NTcyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZiOWNjMGFkZWQyNGViMzU3M2IxNjQ4NjVhMWQ0MTUxYzFjOGQ3NjJkMTE1M2ZmZmI5MGRjZGMyN2ZmODczNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2JEGOfk9bEp69X7rMAyzzqhzW-cqDnbJNolSI2SBrHg">
> More note on this: In IHME, COVID-19 is directly put under the cause of "Respiratory infections and tuberculosis" as a level-3 cause, in parallel to other level-3 causes such as "Lower respiratory infections". Additionally, IHME added a new level-1 cause called "other COVID-19 related outcome" that is not categorized under any of the three level-1 causes. Thus,
> 
> <img alt="Image" width="381" src="https://private-user-images.githubusercontent.com/34465524/411704850-61ad699e-0203-42e6-be44-9787f7fdc450.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkyMTc3NDIsIm5iZiI6MTczOTIxNzQ0MiwicGF0aCI6Ii8zNDQ2NTUyNC80MTE3MDQ4NTAtNjFhZDY5OWUtMDIwMy00MmU2LWJlNDQtOTc4N2Y3ZmRjNDUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjEwVDE5NTcyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlOWI1OTVmZjJkZTVkOTE0YTA2NDRiZWIwNDAyMTE2Mjk5NDQ5YzI1MDdkODE2NjVjYzFlMjQzZmQzOTJjZjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.eTgTKowwPUDLq9s5ECFJa1hfPlMJwf36-N4L5TKZOwo">
> 1. We can probably put "other COVID-19 related outcomes" under "respiratory infection", but this needs a consensus from the team.
> 2. Even if we ignore this newly added level-1 cause, in the updated data, we should see a huge spike in the "Respiratory Infection" in IFs. We need to investigate how it will drive further model changes.

My vote: Put it in Respiratory, then have a fellow compare health forecasts in a version with the new data and with the old data.

**Comment by PardeeCenterIFs** [comment]
I assume then that covid deaths are still not in the model, but I thought they were, the table that we use for initialization of mortality is HealthDetailedDeathsCtry, and I was trying to check when was that last updated or any available info on that so I went to the wiki looking for info, but couldn't find any, am I looking at the wrong place?

https://pardeewiki.du.edu/index.php?title=Data

**Comment by quciet** [comment]
> I assume then that covid deaths are still not in the model, but I thought they were, the table that we use for initialization of mortality is HealthDetailedDeathsCtry, and I was trying to check when was that last updated or any available info on that so I went to the wiki looking for info, but couldn't find any, am I looking at the wrong place?
> 
> https://pardeewiki.du.edu/index.php?title=Data

Here are pages related to IHME data update- 
https://pardeewiki.du.edu/index.php?title=IFs.db#HealthDetailedDeathsCtry
https://pardeewiki.du.edu//index.php?title=IHME

We updated them last year. 

As I mentioned this morning, I remember working with data related to covid mortality. I then checked my working folder, and it seems it was near the end of 2020 when we were working with the African Union. At that time, IHME had a standalone project that forecasted COVID deaths across the world. Although we had questions about their projections, I took their data and made some scenarios that put those COVID deaths under the variable hlcomaddprev. I think at one point we even packaged these scenarios into the installation version of IFs. 

After the project, and once the world started to open up, we stopped keeping track of this. 

So, to conclude, we had it in IFs under the scenarios for a while due to a project need. But we never actually had the data. Last year's update of IHME was using IHME GBD 2019, and these COVID deaths are now available under the IHME GBD 2021 release .

**Comment by PardeeCenterIFs** [comment]
Is there a link from the DATA page (https://pardeewiki.du.edu/index.php?title=Data) to go to this ifs.db page (https://pardeewiki.du.edu/index.php?title=IFs.db) that you sent, and includes this IHME update?

I see we updated the DetailedDeath table in February 2024, and see the scenarios with hlcomaddprev. I guess that means the DetailedDeaths file currently does not have all deaths in the world, and the update in question will have them, unless we leave out Covid. Currently I see 59.32 Million global deaths in 2020 in the base case, which doesn't use the hlcomaddprev parameter.

I'm ok with putting those Covid Deaths under Respiratory Infections and test it as Collin suggests.

Also the content of DetailedDeathCtry is Deaths per Thousand, right? if that's correct probably should add it to the wiki.

**Comment by quciet** [comment]
I'm moving this to priority 1 because the data (2021GBD) are already in IFs, I just need to make sure with a new base (2022 instead of 2020), no further changes will be needed.
